OrbitQt/main.cpp

```diff
@@ -314,6 +314,7 @@ int main(int argc, char* argv[]) {
 #endif
 
     QApplication app(argc, argv);
+    QApplication::setOrganizationName("The Orbit Authors");
```

Comment:
Is that changing where QSettings stores its values?

Reply:
Apparently on Windows it is necessary to set the Organization name, or QSettings will not work (when used without arguments). 
Besides that, yes, it will change, when the arguments are removed from the other 2 occurrences of QSettings. Before we used this for QSettings: "The Orbit Authors" ; "Orbit Profiler". 
Since the application name is set to "orbitprofiler" in the next line. It will change to "The Orbit Authors" ; "orbitprofiler". 

Does that answer the question? 

I will write something in chat, to see if anyone has a problem with losing the settings about remembering not to show the warnings

Reply:
Thanks for figuring that out. Of course that makes a lot of sense. you need that organization name for the registry.
Thanks for fixing that!

Yes, that answers my question!